packages/filesystem/src/browser/file-tree/file-tree.ts

```diff
@@ -96,6 +95,13 @@ export namespace FileStatNode {
     export function is(node: TreeNode | undefined): node is FileStatNode {
         return !!node && 'fileStat' in node;
     }
+
+    export function getUri(node: TreeNode | undefined): string {
+        if (!node) {
+            return '';
```

Comment:
an empty string is not an uri, you should return `undefined` and a client should decide what to do